### ISSUE TITLE
main text box showing job description and highlighted ableist language is missing ARIA label

### ISSUE THREAD

**Issue by dianalam** [USDepartmentofLabor/ableist-language-detector-web#3]

**Comment by SamPotasz** [comment]
@dianalam, are you seeing this as an error somewhere? This block of text is just text so doesn't need a label

**Comment by dianalam** [comment]
@SamPotasz I didn't see this as an error, but I wasn't sure if the screenreader would be able to access the text and highlighted terms within the text box without the label. But if I'm wrong, we can close this issue!
